soul-web/src/test/java/org/dromara/soul/web/DubboMultiParameterResolveServiceImplTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dromara.soul.web;
+
+import lombok.Data;
+import org.apache.commons.lang3.tuple.Pair;
+import org.dromara.soul.web.dubbo.DubboMultiParameterResolveServiceImpl;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Test cases for WebSocketParamFilter.
+ *
+ * @author candyYu
+ */
+@RunWith(MockitoJUnitRunner.class)
+public final class DubboMultiParameterResolveServiceImplTest extends DubboMultiParameterResolveServiceImpl {
```

Comment:
Why do we extends `DubboMultiParameterResolveServiceImpl ` here?

Reply:
i don't get it ,shoud i create a DubboMultiParameterResolveServiceImpl

Reply:
yeah. I suggest you new/mock the instance.

Reply:
i had new the instance with mock, please recheck